compositor_render/src/scene/view_component/layout.rs

```diff
@@ -72,12 +72,18 @@ impl ViewComponentParam {
         let children: Vec<_> = children
             .iter_mut()
             .map(|child| {
-                let position = match child {
-                    StatefulComponent::Layout(layout) => layout.position(pts),
-                    ref non_layout_component => Position::Static {
-                        width: non_layout_component.width(pts),
-                        height: non_layout_component.height(pts),
-                    },
+                let (position, padding) = match child {
+                    StatefulComponent::Layout(StatefulLayoutComponent::View(view)) => {
```

Comment:
Why do you need to know padding of a child?

Reply:
Not necessary anymore. I removed it